loading.bs

```diff
@@ -522,20 +578,20 @@ the result of the following steps:
     :: the result of [=parsing b3 CBOR headers=] given |headerBytes| and
         |requestUrl|.
 1. If |parsedSignature| [=exchange signature/does not establish cross-origin
-    trust=] for |parsedExchange|, return a failure.
+    trust=] for |parsedExchange|, return `"cert_verification_error"`.
 1. Set |parsedExchange|'s [=exchange/response=]'s [=response/HTTPS state=] to
     either "`deprecated`" or "`modern`".
 
     Note: See <a spec="fetch">HTTP-network fetch</a> for details of this choice.
 1. If |parsedExchange|'s [=exchange/response=]'s [=response/status=] is a
     [=redirect status=] or the [=signed exchange version=] of |parsedExchange|'s
-    [=exchange/response=] is not undefined, return a failure.
+    [=exchange/response=] is not undefined, return `"parse_error"`.
 
     Note: This might simplify the UA's implementation, since it doesn't have to
     handle nested signed exchanges.
 1. [=Read a body=] from |stream| into |parsedExchange|'s [=exchange/response=]
     using |parsedSignature| to check its integrity. If this is a failure, return
-    the failure.
+    `"mi_error"`.
```

Comment:
I'm a little worried this'll get stale if [read a body](https://pr-preview.s3.amazonaws.com/horo-t/webpackage/pull/374.html#reading-a-body) ever grows another kind of error return. Maybe move the definition of the error type into that algorithm?